flake.nix

```diff
@@ -2,21 +2,40 @@
   description = "soxin: opiniated configs for everyone";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs";
+    deploy-rs.url = "github:serokell/deploy-rs";
+    flake-utils-plus.url = "github:gytis-ivaskevicius/flake-utils-plus/v1.1.0";
+    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
Why to we need to provide unstable here? Shouldn't we let it up to the user to choose?

Reply:
Adding it here makes it available to the user without having them wire it up in their own cfg. I thought the same tbh and was debating if it's good or bad and realized one thing: if they don't use it then it won't be fetched or evaluated so what's the harm is adding it.